mvp-spec/publishers-requirements.md

```diff
@@ -41,55 +38,34 @@ sequenceDiagram
 ```
 <!--partial-end-->
 
-## Offer inventory with Prebid Addressability Framework Data
+## Selling ad slots with the Prebid Addressability Framework
 
-PAF doesn't standardize the API of the Ad Server for offering
-inventory because each Ad Server has an existing and specific API. 
-However, to implement PAF, the Ad Server need to implement new features
-in the existing endpoints called by the Publisher website to offer inventory 
-and get back Addressable Contents.
-
-To understand the steps, it is important to overview how to generate the data and
-the relationships between them:
-* A Publisher offers *multiple* placements - one for each for 
-Addressable Contents - via an Ad Server.
-* An Ad Server generates *one* Seed for each Addressable Content
-* An Ad Server generates and sends *one* Transaction Request per Seed and Supplier
-* A Supplier generates and sends *one* Transaction Response per Transaction Request
+### ad slots, Seeds, and Transmissions Requests
 
+The relationships between ad slots, Seeds, and Transmissions are:
+* A Publisher offers *multiple* ad slots per page
+* The Publisher must create *one* Seed for per ad slot
+* The Publisher must send *one* Transmission Request per Seed and SSP
+* An SSP must generate *one* Transmission Response per Transmission Request
 
 ```mermaid
 flowchart LR
-    subgraph AdServer
-        Seed-- 1-n -->TR[Transmission Request]
-    end
-
-    subgraph Publisher
-        Placement-- 1-1 -->Seed
-    end
-
-     subgraph Supplier
-         TR[Transmission Request]-- 1-1 -->TRp[Transmission Response]
-    end
+    
+    Page-- 1-n -->AD[Ad Slot]
+    AD[Ad Slot]-- 1-1 -->Seed
+    Seed-- 1-n -->TR[Transmission Request]
+    TR[Transmission Request]-- 1-1 -->TRp[Transmission Response]
+    
 ```
 
-### Step 1: Require the Prebid SSO Cookies
-
-The Ad Server API has to be extended to require the Prebid SSO cookies of 
-the Publisher website for a given user in addition to the inventory information. 
-Those cookies contain the Prebid SSO Data. As a reminder, 
-Prebid SSO Data are Pseudonymous-Identifiers and Preferences of the user.
+### Step 1: Access PAF User Id and Preferences
 
-```mermaid
-flowchart LR
-    Publisher-- Send Placements info <br /> with Prebid SSO Cookies -->AdServer[Ad Server]
-```
+*To be added.*
 
-### Step 2: Deserialize the Prebid SSO Data
 
-The Ad Server can then deserialize the cookie for having the Prebid SSO Data.
+### Step 2: Deserialize the User Id and Preferences
 
-Here is the structure of a Pseudonymous-Identifier:
+The structure of the User Id is:
 
 <!--partial-begin { "files": [ "identifier-table.md" ] } -->
```

Comment:
This should simply link to a file in `/model/`
- for consistency
- because it's more up to date (this partial is obsolete)

(probably For Another Review, I think @RomainLofaso is already working on it)

Reply:
Can I leave it to @RomainLofaso in another PR?